src/components/Book.js

```diff
@@ -17,9 +17,7 @@ const Book = ({ item, removeBook }) => (
 );
 
 Book.propTypes = {
-  item: PropTypes.string.isRequired,
-  title: PropTypes.string.isRequired,
-  category: PropTypes.string.isRequired,
+  item: PropTypes.shape({}).isRequired,
   removeBook: PropTypes.func.isRequired,
 };
```

Comment:
- [ ] [REQUIRED] I got an error when open your APP on my local machine. It will be nice if you fix it.
![1](https://user-images.githubusercontent.com/58552748/132440636-acdbd487-ecf3-46e0-b6f3-77cf0b3d32b9.png)

Reply:
Can you explain me how to fix that, please?

Reply:
You can do something like this: (it is just an example, feel free to update it based on your props)

Book.propTypes = {
  book: PropTypes.string.isRequired,
  book: PropTypes.shape({
    id: PropTypes.number.isRequired,
    title: PropTypes.string.isRequired,
    category: PropTypes.string.isRequired,
  }),
  removeBook: PropTypes.func.isRequired,
};

Book.defaultProps = {
  book: {
    id: Math.floor(Math.random() * 100),
    title: 'not defined',
    category: 'not defined',
  },
};

Reply:
Thank you! I didn't know that one!